heron/scheduler-core/tests/java/com/twitter/heron/scheduler/dryrun/UpdateDryRunRenderTest.java

```diff
@@ -0,0 +1,156 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.scheduler.dryrun;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.google.common.io.Files;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import com.twitter.heron.api.generated.TopologyAPI;
+import com.twitter.heron.common.basics.ByteAmount;
+import com.twitter.heron.spi.common.Config;
+import com.twitter.heron.spi.packing.InstanceId;
+import com.twitter.heron.spi.packing.PackingPlan;
+// SUPPRESS CHECKSTYLE AvoidStarImport
+import com.twitter.heron.spi.packing.PackingPlan.*;
+import com.twitter.heron.spi.packing.Resource;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(TopologyAPI.Topology.class)
+public class UpdateDryRunRenderTest {
+
+  private PackingPlan originalPlan;
+  private PackingPlan newPlanA;
+  private PackingPlan newPlanB;
+
+  @Before
+  public void setUp() throws Exception {
```

Comment:
See comment above. We should avoid all this boilerplate setup code in favor of using shared test utility helpers